locations/spiders/thecheesecakefactory.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+from scrapy.spiders import SitemapSpider
+
+from locations.hours import OpeningHours
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class TheCheesecakeFactorySpider(SitemapSpider, StructuredDataSpider):
+    download_delay = 0.1
+    name = "thecheesecakefactory"
+    item_attributes = {"brand": "The Cheesecake Factory", "brand_wikidata": "Q1045842"}
+    allowed_domains = [
+        "www.thecheesecakefactory.com",
+        "locations.thecheesecakefactory.com",
+    ]
+    sitemap_urls = ["https://locations.thecheesecakefactory.com/robots.txt"]
+    sitemap_rules = [("", "parse_sd")]
```

Comment:
If we checkout the sitemap, it includes pages for the [states](https://locations.thecheesecakefactory.com/ok/) and the [cities](https://locations.thecheesecakefactory.com/ok/tulsa/), we don't need to fetch them, the following filter only matches on the store pages.

```suggestion
    sitemap_rules = [(r"https:\/\/locations\.thecheesecakefactory\.com\/.+\.html", "parse_sd")]
```

Reply:
Thanks I've added it in #4095